include/uv.h

```diff
@@ -682,6 +682,18 @@ enum uv_udp_flags {
    * This flag is no-op on platforms other than Linux.
    */
   UV_UDP_LINUX_RECVERR = 32,
+  /*
+   * Indicates if SO_REUSEPORT will be set when binding the handle.
```

Comment:
Please update the comment in `UV_UDP_REUSEADDR` since it does mention REUSEPORT on some systems.

Reply:
Update the comment with what? About what platforms **will not** be set with `SO_REUSEPORT` when specifying `UV_UDP_REUSEADDR`?

Reply:
Yes please.

Reply:
Done.